app/javascript/packages/document-capture/components/review-issues-step.tsx

```diff
@@ -92,37 +104,149 @@ function ReviewIssuesStep({
+          actionText={t('idv.failure.button.warning')}
+          actionOnClick={onWarningPageDismissed}
+          location="doc_auth_review_issues"
+          remainingAttempts={remainingAttempts}
+          troubleshootingOptions={
+            <DocumentCaptureTroubleshootingOptions
+              location="post_submission_warning"
+              showAlternativeProofingOptions={!isFailedResult}
+              heading={t('components.troubleshooting_options.ipp_heading')}
+            />
+          }
+        >
+          {!!unknownFieldErrors &&
+            unknownFieldErrors
+              .filter((error) => !['front', 'back'].includes(error.field!))
+              .map(({ error }) => <p key={error.message}>{error.message}</p>)}
+
+          {remainingAttempts <= DISPLAY_ATTEMPTS && (
+            <p>
+              <strong>
+                {remainingAttempts === 1
+                  ? t('idv.failure.attempts.one')
+                  : t('idv.failure.attempts.other', { count: remainingAttempts })}
+              </strong>
+            </p>
+          )}
+        </Warning>
+      );
+    }
+    if (inPersonCtaVariantActive === 'in_person_variant_a') {
+      trackEvent('IdV: IPP CTA Variant A');
+      return (
+        <Warning
+          heading={t('errors.doc_auth.throttled_heading')}
+          actionText={t('idv.failure.button.warning_variant')}
+          actionOnClick={onWarningPageDismissed}
+          location="doc_auth_review_issues"
+          remainingAttempts={remainingAttempts}
+          troubleshootingOptions={
+            <DocumentCaptureTroubleshootingOptions
+              location="post_submission_warning"
+              showAlternativeProofingOptions={!isFailedResult}
+              heading={t('components.troubleshooting_options.ipp_heading')}
+              altInPersonCta={t('in_person_proofing.headings.cta_variant')}
+              altInPersonPrompt={t('in_person_proofing.body.cta.prompt_detail_a')}
+              altInPersonCtaButtonText={t('in_person_proofing.body.cta.button_variant')}
+            />
+          }
+        >
+          <h2>{t('errors.doc_auth.throttled_subheading')}</h2>
+          {!!unknownFieldErrors &&
+            unknownFieldErrors
+              .filter((error) => !['front', 'back'].includes(error.field!))
+              .map(({ error }) => <p key={error.message}>{error.message}</p>)}
+
+          {remainingAttempts <= DISPLAY_ATTEMPTS && (
+            <p>
               {remainingAttempts === 1
-                ? t('idv.failure.attempts.one')
-                : t('idv.failure.attempts.other', { count: remainingAttempts })}
-            </strong>
-          </p>
-        )}
-      </Warning>
-    );
+                ? formatWithStrongNoWrap(t('idv.failure.attempts.one_variant_a_html'))
+                : formatWithStrongNoWrap(
+                    t('idv.failure.attempts.other_variant_a_html', { count: remainingAttempts }),
+                  )}
```

Comment:
#7667 updates to use default pluralization. I would suggest structuring the variants as a hierarchy.

e.g.

```yaml
en:
  idv:
    failure:
      attempts:
        default:
          one: ...
          other: ...
        variant_a:
          one: ...
          other: ...
        variant_b:
          one: ...
          other: ...
```

Reply:
Will be in a separate PR from @tomas-nava

Reply:
Here's the ticket for that work: **[LG-8756](https://cm-jira.usa.gov/browse/LG-8756)**